bucket/oss-cad-suite-nightly.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "2022-03-31",
+    "description": "Open source digital design and verification tools. Includes tools for RTL synthesis, formal hardware verification, place & route, FPGA programming, and testing with support for HDLs like Verilog, Migen and Amaranth.",
+    "homepage": "https://github.com/YosysHQ/oss-cad-suite-build",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/YosysHQ/oss-cad-suite-build/releases/download/2022-03-31/oss-cad-suite-windows-x64-20220331.exe#/dl.7z",
+            "hash": "de3c9525aafd8a0981b31c92a2543744e49108ad79751a028ecb460baf891cf4"
+        }
+    },
+    "extract_dir": "oss-cad-suite",
+    "pre_install": "Set-Content -Path \"$dir\\start.bat\" -Value \"@cmd /k $dir\\environment.bat\"",
```

Comment:
As I said, this file already exists. No need to create it.

Reply:
If my pre_install script is not added then `start.bat` won't be able to find `environment.bat` unless run from `$dir`